homeassistant/components/plugwise/number.py

```diff
@@ -0,0 +1,103 @@
+"""Number platform for Plugwise integration."""
+from __future__ import annotations
+
+from collections.abc import Awaitable, Callable
+from dataclasses import dataclass
+from typing import Any
+
+from plugwise import Smile
+
+from homeassistant.components.number import (
+    NumberDeviceClass,
+    NumberEntity,
+    NumberEntityDescription,
+    NumberMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import PlugwiseDataUpdateCoordinator
+from .entity import PlugwiseEntity
+
+
+@dataclass
+class PlugwiseEntityDescriptionMixin:
+    """Mixin values for Plugwse entities."""
+
+    command: Callable[[Smile, float], Awaitable[Any]]
```

Comment:
Would it make sense to use `Awaitable[None]` or will there be cases where there is a return value?
```suggestion
    command: Callable[[Smile, float], Awaitable[None]]
```

Reply:
What does this `Any` or `None` relate to?
Does it relate to what the `api.set_max_boiler_temperature()`-call returns?
If yes, then None is correct as the function does not return anything.

Reply:
Suggestion was implemented.